docs/reference/2-ods-api/how-to-guides/how-to-add-profiles-to-the-ed-fi-ods-api.md

```diff
@@ -26,20 +16,39 @@ Organizations, Profile assignments, and Profile definitions.
 
 The tables related to this process are shown below.
 
-![Table Relationships](../../../../static/img/reference/ods-api/image-2023-4-5_22-34-39.png)
+```mermaid
+erDiagram
+    Profiles ||--o{ ProfileApplications : has
+    Applications ||--o{ ProfileApplications : has
+
+    Profiles {
+        int ProfileId PK
+        varchar ProfileName
+        varchar ProfileDefinition
+    }
+
+    ProfileApplications {
+        int Profile_ProfileId PK
+        int Application_ApplicationId PK
+    }
+
+    Applications {
+        int ApplicationId PK
+        varchar ApplicationName
+        int Vendor_VendorId
+        varchar ClaimSetName
+        int OdsInstance_OdsInstanceId
+        varchar OperationalContextUri
+    }
```

Comment:
Fun thing about working on all of these mermaid charts - I can now read them in plain text 😆 .

Reply:
Yea these have been really straightforward to review. Adding examples to our documentation.